meta/.phpstorm.meta.php

```diff
@@ -653,6 +653,43 @@ function exitPoint($functionReference) {
     exitPoint(\wp_die());
     exitPoint(\dd());
 
+    /**
+     * @param callable $callable Class, Method or function call
+     * @see callback()
+     * @see arg()
+     * @return mixed
+     */
+    function registerCallbackTypes($callable) {
+      return "registerCallbackTypes $callable";
+    }
+
+    /**
+     * description of the callback arguments
+     * @param mixed $args
+     * @see arg()
+     * @return mixed
+     */
+    function callback(...$args) {
+      return "callback ${$args}";
+    }
+
+    /**
+     * argument type #$argNum of the caller callback
```

Comment:
This needs to be a type of an _element_, not the type of the argument (argument to array_map is array, and the callbacks accept an element of an array)

Reply:
Yes, it is, thanks.